README.md

```diff
@@ -131,6 +132,29 @@ archive { '/tmp/test100k.db':
 }
 ```
 
+### Puppet URL
+
+Currently there isn't support for using ```puppet:\\``` style url in the source parameter.  It is still possible, but it would be done in two steps:
+
+```puppet
+$docs_filename = 'help.tar.gz'
+$docs_gz_path  = "/tmp/${docs_filename}"
+$homedir = '/home/myuser/'
```

Comment:
Before getting into the specifics of usage, describe what the goal is, i.e. to place the file `help.tar.gz` from the `profile` module's `files` directory at `/tmp/help.tar.gz`, and then expand that file in the user's home directory, `/home/myuser`. I assume that is correct, anyway.